### ISSUE TITLE
About your architecture

### ISSUE THREAD

**Issue by csjfwang** [delta-onera/segnet_pytorch#2]
hi @aplyer @aboulch @blesaux @achanhon ,

may i ask your architecture of segnet? i just read the caffe code.
i don't know which one is it, segnet? segnet_basic? or bayesian segnet?

thx a lot!

**Comment by aboulch** [comment]
Hello,
It is SegNet, including batchnorm.
There is now an updated version the delta-tb. You would also find a UNet.
Best.

**Comment by csjfwang** [comment]
OK @aboulch  , I have learnt a lot from your code.
thx a ton!

**Comment by csjfwang** [comment]
@aboulch
sorry, another problem is that i tried to train this segnet for semantic segmentation,
but, the loss cannot converge, so i wanna ask if you have tried this code on segmentation task.

thx a lot!
